test/unit/org/apache/cassandra/tools/nodetool/stats/NodetoolTableStatsTest.java

```diff
@@ -237,4 +237,13 @@ public void testTopArg()
         tool.assertCleanStdErr();
         assertEquals(1, tool.getExitCode());
     }
+
+    @Test
+    public void testStatusArg()
```

Comment:
Thanks for adding a test. I'm wondering if this should be a new package in nodetool tests since we're testing `status` and not `stats`.

Reply:
+1